packages/amplify_authenticator/lib/src/enums/gender.dart

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+*/
+
 enum Gender {
   male,
   female,
```

Comment:
maybe add 
`genderNonbinary`, `preferNotToSay`

Reply:
Okay, will probably update this later since it's not even being used for anything yet. We just have gender as a text field still.